wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -21,14 +21,12 @@
     - `action_icon` - icon for the 'action' button, default is 'plus'
     - `base_actions` - base actions to appear before the main action
     - `extra_actions` - extra action buttons for the header. This is the HTML to be used for the extra buttons
-    - `breadcrumb` - Custom breadcrumbs content as a variable, displayed in place of breadcrumb block before the header content
     - `heading` - Replaces the header `h1` completely with custom variable content
     - `search` - An alternative search content variable displayed if provided and search_url url is not present
     - `extra_rows` - Shown at the end of the header container
 
 {% endcomment %}
 <header class="{% classnames "w-header" classname merged|yesno:"w-header--merged," search_form|yesno:"w-header--hasform," %}">
-    {% block breadcrumb %}{{ breadcrumb }}{% endblock %}
```

Comment:
No longer used

Reply:
The `breadcrumb` block was added in #4029, but then we got rid of ModelAdmin breadcrumbs entirely in #8743, but the block wasn't removed from the `index.html` template (you can see that PR doesn't change `modeladmin/index.html`. Then when #9993 was added, the `breadcrumb` block was retained as an empty fragment.

The same is true for the package, see the following code search:
https://github.com/search?q=repo%3Awagtail-nest%2Fwagtail-modeladmin%20breadcrumb&type=code